CRM/Core/Payment/Manual.php

```diff
@@ -17,6 +17,16 @@
 class CRM_Core_Payment_Manual extends CRM_Core_Payment {
 
   protected $result;
+  protected $_supports = [
```

Comment:
I think this approach is generally fine - we have been moving away from the underscore in class property names though - see the property above.